packages/ui/apps/origin-ui/src/hooks/useInterceptors.ts

```diff
@@ -0,0 +1,26 @@
+import { getAuthenticationToken } from '@energyweb/origin-ui-shared-state';
+import axios from 'axios';
+
+declare global {
+  interface Window {
+    config: {
+      BACKEND_URL: string;
+      BLOCKCHAIN_EXPLORER_URL: string;
+      SUPPORTED_NETWORK_IDS: string;
```

Comment:
Do we need to add other variables here such as:
NX_GOOGLE_MAPS_API_KEY
NX_EXCHANGE_WALLET_PUB
NX_REGISTRATION_MESSAGE_TO_SIGN
NX_SMART_METER_ID

Reply:
No, because then they will be publicly available and we don't want this to happen